docs/schemas/Library-Minimal-Metadata/Library Minimal Metadata.csv

```diff
@@ -0,0 +1,52 @@
+dissociated cell sample label,65e2c7da-9eb4-45b2-8ccb-d69ef9785ee2,dissociated cell sample name,,"Name of a collection of dissociated cells or nuclei derived from dissociation of a tissue sample.
+",,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Dissociated cell oligo tag name,184abbaf-baff-4b5f-b51e-dd38de6006af,,,"Name of oligo used in cell plexing.  The oligo will tag allow separate dissociated cell samples to be combined downstream in the barcoded cell sample.  The oligo name is associated with a sequence in a lookup table.  This sequence will be needed to during analysis, after alignment, to associate reads with parent dissociated cell sample.",,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Enriched cell sample container name,5ace37aa-85d6-4493-909e-8fc221ec2609,,,Name of container (strip or tube or plate) of the enriched_cell_prep.  This container could contain 1 or more enriched_cell_samples.,,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Enriched cell sample name,bb3fc701-23a7-45c1-890d-7471730e0ec1,,,Name of collection of enriched cells or nuclei after enrichment process (usually via FACS using the Enrichment Plan) applied to dissociated_cell_sample.,,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Enrichment population,875f1c70-f5aa-45e3-94b9-5e482f6c4830,,,Actual percentage of cells as a result of using set of fluorescent marker label(s) to enrich dissociated_cell_sample with desired mix of cell populations.  This plan can also be used to describe 'No FACS' where no enrichment was performed.  This is a property of enriched_cell_prep_container. ,,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library avg size bp,f851eba9-56d1-4472-9d0c-d7f8bc33000a,,,"Average size of the library in terms of base pairs.  This is used to calculate the molarity before pooling and sequencing.
+",,,TRUE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library method,7b60d59e-fdd7-4b27-a2d4-cae9b69103a6,library chemistry method,,"Standardized nomenclature to describe the library method used.  This specifies the alignment method required for the library.  For example, 10xV3.1 (for RNASeq single assay), 10xMult-GEX (for RNASeq multiome assay), and 10xMult-ATAC (for ATACSeq multiome assay)",,,TRUE,,ValueSet,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Library concentration nm,90805b3f-f380-4f23-b159-e7eaa0c8f052,,,Concentration of library in terms of nM (nMol/L).  Number of molecules is needed for accurate pooling of the libraries and for generating the number of target reads/cell in sequencing.,,,FALSE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```

Comment:
Library concentration nm: expected data type -> float
data example provided in green table: 123.6382113

```suggestion
Library concentration nm,90805b3f-f380-4f23-b159-e7eaa0c8f052,,,Concentration of library in terms of nM (nMol/L).  Number of molecules is needed for accurate pooling of the libraries and for generating the number of target reads/cell in sequencing.,,,FALSE,,Float,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```